src/drivers/dw/dma.c

```diff
@@ -1028,12 +1028,8 @@ static int dw_dma_copy(struct dma *dma, unsigned int channel, int bytes,
 		return -EINVAL;
 	}
 
-	/* do nothing on preload */
-	if (flags & DMA_COPY_PRELOAD)
```

Comment:
@tlauda I see this is just revert 4549747e37e60db458ac75c93dfd5531afba37a4
Could you give more explanation about why we need this at first time and why do not need it here?
It seems we try ignore the PRELOAD but try to ignore it here again? Maybe PRELOAD and ONESHOT is the same now.